src/main/java/EventScheduler.java

```diff
@@ -0,0 +1,114 @@
+package main.java;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EventScheduler {
+    private ArrayList<Event> eventList;
+    private TaskToEvent converter;
+    private TodoList todoList;
+    private GapFinder gapFinder;
+
+    public EventScheduler(TaskToEvent obj, GapFinder gapFinder){
+        this.eventList = new ArrayList<>();
+        this.converter = obj;
+        this.gapFinder = gapFinder;
+    }
+
+    public void addEvents(Event event){
+        this.eventList.add(event);
+    }
+
+    /**
+     *
+     * @param event event to be removed
+     * @return if the event has been removed
+     */
+    public boolean removeEvents(Event event){
+        if (this.eventList.contains(event)) {
+            this.eventList.remove(event);
+            return true;
+        }
+        return false;
+    }
+
+    public void addTaskToTodoList(){
```

Comment:
It seems that tasks might be added to todoList repeatedly.
Also, can add some documentation for this method sine the name of function does not describe what it does well (actually it's adding tasks inside events)